cargo-fuzz/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM rust:1.44-buster
+RUN cargo install cargo-fuzz && \
+    rustup install nightly && \
+    rustup default nightly
+ 
+LABEL LANGUAGES "rust"
+LABEL FUZZER "cargo-fuzz"
```

Comment:
I'm hoping to use this label as "fuzzerclass", which in this case and for go-fuzz would be libfuzzer. perhaps should rename the field to fuzzerclass?

Reply:
Done!